lib/qa/authorities/web_service_base.rb

```diff
@@ -32,6 +32,7 @@ def get_json(url)
     # @param url [String]
     # @return [Faraday::Response]
     def response(url)
+      Faraday::Utils.default_params_encoder = Faraday::FlatParamsEncoder
```

Comment:
Isn't it already using the `FlatParamsEncoder` in the response method in `Loc::GenericAuthority`?  I'm concerned this would change the behavior for other users of WebServiceBase that do not want `FlatParamsEncoder`